source/includes/gridfs/gridfs.py

```diff
@@ -1,41 +1,39 @@
-# start create bucket
-const db = client.db(dbName);
-const bucket = new mongodb.GridFSBucket(db);
-# end create bucket
+    # start create bucket
+    client = MongoClient("<connection string>")
+    db = client["db"]
+    bucket = gridfs.GridFSBucket(db)
+    # end create bucket
```

Comment:
Q: why is the code in this file indented?

Reply:
Fixed